docs/design/VehicleDynamics.md

```diff
@@ -0,0 +1,71 @@
+# Vehicle Dynamics
+
+In this section, we describe the vehicle model which is implemented in scenario_simulator_v2.
+
+
+## Vehicle Model Types
+
+Pose is calculated by bicycle kinematics model.
+
+Steering and velocity models are implemented 3 types of models.
+
+### IDEAL_STEER
+
+Uses velocity command. The steering and velocity vary ideally as commanded.
+
+### DELAY_STEER
+
+Uses velocity command. The steering and velocity vary following a first-order delay model.
+
+### DELAY_STEER_ACC
+
+Uses acceleration command. The steering and acceleration vary following a first-order delay model.
+
+## Vehicle Model Parameters
+| Name                 | Type   | Description                                          | IDEAL STEER | DELAY STEER | DELAY STEER 　 ACC | Default value | unit    |
```

Comment:
```suggestion
| Name                 | Type   | Description                                          | IDEAL_STEER | DELAY_STEER | DELAY_STEER_ACC | Default value | unit    |
```
Is it intentional? (Warning: This suggestion might break the column width)

Reply:
Thank you.
I fiexed.

Reply:
Thank you I fiexed in https://github.com/tier4/scenario_simulator_v2/pull/540/commits/a3c0c2ed4ebabfad7c80aed6bf46f0dbc00d9397